mmn/client.go

```diff
@@ -91,10 +91,17 @@ func (m *client) UpdateBeneficiary(data *UpdateBeneficiaryRequest) error {
 	if err != nil {
 		return err
 	}
+	defer resp.Body.Close()
+
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
 		return err
 	}
+
+	if resp.StatusCode < 199 || resp.StatusCode > 299 {
```

Comment:
```suggestion
	if resp.StatusCode < 200 || resp.StatusCode > 299 {
```

Reply:
Don't want 199 to be considered OK status

Reply:
oops